File: src/main/org/nlogo/api/ViewSettings.java
package org.nlogo.api;

public interface ViewSettings
{
	int fontSize() ;
	double patchSize() ;
	double viewWidth() ;
	double viewHeight() ;
	Perspective perspective() ;
    double viewOffsetX() ;
	double viewOffsetY() ;
    boolean drawSpotlight() ;
	boolean renderPerspective() ;
	boolean isHeadless() ;
}

File: src/main/org/nlogo/api/ViewSettings.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.api

trait ViewSettings {
  def fontSize: Int
  def patchSize: Double
  def viewWidth: Double
  def viewHeight: Double
  def perspective: Perspective
  def viewOffsetX: Double
  def viewOffsetY: Double
  def drawSpotlight: Boolean
  def renderPerspective: Boolean
  def isHeadless: Boolean
}
